app/subscriber/yarn.lock

```diff
@@ -14740,9 +14729,9 @@ __metadata:
   languageName: node
   linkType: hard
 
-"tno-core@file:./tno-core-0.1.134.tgz::locator=mmi-subscriber-app%40workspace%3A.":
-  version: 0.1.134
-  resolution: "tno-core@file:./tno-core-0.1.134.tgz::locator=mmi-subscriber-app%40workspace%3A."
+"tno-core@file:./tno-core-0.1.136.tgz::locator=mmi-subscriber-app%40workspace%3A.":
```

Comment:
You didn't correctly run `yarn install` to fix this lock.  You manually changed it.